pkg/dmesg/testdata/dmesg.peermem.log.0

```diff
@@ -0,0 +1,7 @@
+kern  :err   : 2025-02-10T16:28:06,502716+00:00 nvidia-peermem nv_get_p2p_free_callback:127 ERROR detected invalid context, skipping further processing
+kern  :err   : 2025-02-10T16:28:06,514050+00:00 nvidia-peermem nv_get_p2p_free_callback:127 ERROR detected invalid context, skipping further processing
+kern  :err   : 2025-02-10T16:28:06,525389+00:00 nvidia-peermem nv_get_p2p_free_callback:127 ERROR detected invalid context, skipping further processing
+kern  :err   : 2025-02-10T16:28:06,536726+00:00 nvidia-peermem nv_get_p2p_free_callback:127 ERROR detected invalid context, skipping further processing
+kern  :err   : 2025-02-10T16:28:06,548051+00:00 nvidia-peermem nv_get_p2p_free_callback:127 ERROR detected invalid context, skipping further processing
+kern  :err   : 2025-02-10T16:28:06,559449+00:00 nvidia-peermem nv_get_p2p_free_callback:127 ERROR detected invalid context, skipping further processing
+kern  :err   : 2025-02-10T16:28:06,570939+00:00 nvidia-peermem nv_get_p2p_free_callback:127 ERROR detected invalid context, skipping further processing
```

Comment:
We should add some different logs here, you can check if we get these logs

Reply:
done